mapdata/WarcraftLegacies/AbilityData/Skin/1195847745.json

```diff
@@ -19,12 +19,6 @@
       "Id": 1835101793,
       "Type": 3,
       "Value": "Spell Shield Spell Book"
```

Comment:
As discussed, let's get this Spell Shield using the new button position trick instead of the old Spell Book trick.